### DISCUSSION TITLE
Running program which uses DLL's

### DISCUSSION THREAD

**Discussion by PH5HP** [otya128/winevdm#1375] [General]
Hello,

I am trying to run a program calles Winpack. It is a 16 bit program which purpose is packet radio. I am a Hamradio Operator. This program has been programmed in Visual Basic 3 and it uses several DLL's like: 

- Cmdialog.vbx
- G4IDE7PL.dll
- Mci.vbx
- Mscomm.vbx
- Threed.vbx
- VBRUN300.dll
- VBOA300.dll
- IDEAGW.dll

Every time I try to run this program I get a message that CMDIALOG.VBX is not found in the SYSTEM directory. I did copy all files (CMDIALOG.VBX included in the SYSTEM directory of WINEDVM. In fact I tried all directories within the WINEDVM directory structure.

So I am not sure how I can solve this problem. I really want to use this program again. Is there someone who can help me and provide me with a solution?

Kind regards,
Henk - PH5HP - fileserver.ph5hp.nl (if my main computer is online)

**Comment by cracyc** [comment]
Is Cmdialo.vbx a typo?  Anyway, try putting the dlls into the same dir as the program.

**Reply by PH5HP** [reply]
Yes, it was a typo .. I corrected it in the original message.

I tried and it did not work. Thank you for your suggestion.

Kind regards,
Henk - PH5HP

**Comment by cracyc** [comment]
Seems to work for me.  Can you make a trace?

**Reply by PH5HP** [reply]
Hi,
TNX for your reply. How do I make a trace?

Kind regards,
Henk - https://fileserver.ph5hp.nl

**Reply by cracyc** [reply]
https://github.com/otya128/winevdm/blob/master/.github/ISSUE_TEMPLATE/bug_report.md

**Reply by PH5HP** [reply]
Hi,
Thank you ...

Installing the latest version solved my problem. Winpack is Running.

Kind regards,
Henk - https://fileserver.ph5hp.nl (only if my main PC is running) for Hamradio Stuff

**Comment by PH5HP** [comment]
Hi,

It have it working now, but another question:

Is it possible to have a DDE connection between 2 programs both started with WineDVM within Windows 10 64-bits?

Kind regards,
Henk

**Comment by cracyc** [comment]
Yes, what are you trying to do?

**Reply by PH5HP** [reply]
Hi,

Thanks for the reply. I have 2 programs which I want to connect through DDE. Both programs are started using WinDVM. It did not work from the start, but installing the latest build of WinDVM solved to problem somehow.

It is working now. See my screenshot below. The  program on the left side of the screenshot has a DDE link with the program on the right side. The program on the right side is the DDE host. Seems to work fine now. The program on the right side is also the program I was writing about with the DLL's which had be available to load by Winpack.

Kind regards,
Henk - https://fileserver.ph5hp.nl
![image](https://github.com/otya128/winevdm/assets/140949730/39790dcd-d1f5-4cb1-9369-7177e91dec3d)
